docs/source/config-file.mdx

```diff
@@ -44,19 +44,50 @@ These fields are under the top-level `graphos` key and define your GraphOS graph
 | `apollo_registry_url`     | `URL`    |         | The URL to use for Apollo's registry                                                                            |
 | `apollo_uplink_endpoints` | `URL`    |         | List of uplink URL overrides. You can also provide this with the `APOLLO_UPLINK_ENDPOINTS` environment variable |
 
-### Forward Headers
+### Headers
+
+The `headers` option allows you to specify a list of hard-coded headers and values to include in all GraphQL requests.
```

Comment:
Can we clarify that these static headers are attached to every single GraphQL request, so it's clearer this feature is different from header forwarding?

Reply:
Header forwarding is _not_ attached to every single GraphQL request?

Reply:
Hmm, how should I put this? 🤔

The distinction is this: `headers` is applied at the server level and is static in that values are fixed when the server starts. In contrast, `forward_headers` operates at the request level and are dynamic in that values are determined at call time.

Not sure if this makes sense. 🥲